### PR TITLE
[oracle] Remove oracle prevote

### PR DESCRIPTION
**Pull request by udpatil**
Still testing with localsei but  here's the PR that removes the prevote functionality

### CODE REVIEW

File: x/oracle/types/genesis.go [hunk 1]
```diff
@@ -10,28 +10,25 @@ import (
...
 }
 
 // DefaultGenesisState - default GenesisState used by columbus-2
 func DefaultGenesisState() *GenesisState {
 	return &GenesisState{
-		Params:                        DefaultParams(),
-		ExchangeRates:                 []ExchangeRateTuple{},
-		FeederDelegations:             []FeederDelegation{},
-		PenaltyCounters:               []PenaltyCounter{},
-		AggregateExchangeRatePrevotes: []AggregateExchangeRatePrevote{},
```
Comment (LCyson) [inline x/oracle/types/genesis.go:33]: do we need to remove this from the state in the upgrade handler?

**Review by LCyson — APPROVED** [review]

File: x/oracle/types/genesis.go (hunk 1 above)
Comment (udpatil) [reply x/oracle/types/genesis.go:33]: yes good point, I'll make that change

File: x/oracle/keeper/migrations.go [hunk 2]
```diff
@@ -56,3 +56,17 @@ func (m Migrator) Migrate3to4(ctx sdk.Context) error {
...
 	return nil
 }
+
+// Migrate3to4 migrates from version 4 to 5
+func (m Migrator) Migrate4to5(ctx sdk.Context) error {
+	// we remove the prevotes from store in this migration
+	store := ctx.KVStore(m.keeper.storeKey)
+
+	oldPrevoteKey := []byte{0x04}
+	iter := sdk.KVStorePrefixIterator(store, oldPrevoteKey)
```
Comment (LCyson) [inline x/oracle/keeper/migrations.go:66]: this removes every in the oracle store right

**Review by LCyson — APPROVED** [review]

File: x/oracle/keeper/migrations.go (hunk 2 above)
Comment (udpatil) [reply x/oracle/keeper/migrations.go:66]: yeah it iterates over the store and deletes anything thats prefixed by the prevote key

**Pull request merged**